src/homeworks/2024_04/test_03/index.ts

```diff
@@ -1 +1,13 @@
-export const getMinList = () => {};
+const toNumber = (numString: string): number => {
+  return Number(numString);
+};
+
+const getNumber = (input: string): number => {
```

Comment:
E' vero questa funzione ritorna un numero, pero' lo chiamerei `getMin` perche' e' piu' descrittivo.

```suggestion
const getMin = (input: string): number => {
```